source/reflect.h2

```diff
@@ -1371,6 +1372,66 @@ print: (t: meta::type_declaration) =
     std::cout << t.print() << "\n";
 }
 
+//-----------------------------------------------------------------------
+//
+//  regex - creates regular expressions from members
+//
+//  TODO
+//
+regex_gen: (inout t: meta::type_declaration) =
+{
+    has_default: bool = false;
```

Comment:
There are several variables here that are explicitly typed that seem like they could easily be implicitly typed.  This `bool` and several `std::string` vars.

Reply:
I removed the type from bool and regex. But for `name` and `expr` I would rather keep it. 

In addition, I simplified the name handling logic.

 - `name` : `m.name()` returns a string view. The overload of map.operator [] has problems with this.
 - `expr: std::string = m.initializer();` : This one could be removed. But since `m.initializer()` could also return a view in the future, I would rather keep it.